var/spack/repos/builtin/packages/cbtf/package.py

```diff
@@ -72,6 +68,51 @@ class Cbtf(Package):
 
     parallel = False
 
+    build_directory = 'build_cbtf'
+
+    def build_type(self):
+        return 'None'
+
+    def cmake_args(self):
+
+        spec = self.spec
+
+        # Boost_NO_SYSTEM_PATHS  Set to TRUE to suppress searching
+        # in system paths (or other locations outside of BOOST_ROOT
+        # or BOOST_INCLUDEDIR).  Useful when specifying BOOST_ROOT.
+        # Defaults to OFF.
+
+        if '+runtime' in spec:
+            # Install message tag include file for use in Intel MIC
+            # cbtf-krell build
+            # FIXME
+            cmake_args = [
+                '-DBoost_NO_SYSTEM_PATHS=TRUE',
+                '-DXERCESC_DIR=%s'         % spec['xerces-c'].prefix,
+                '-DBOOST_ROOT=%s'          % spec['boost'].prefix,
+                '-DMRNET_DIR=%s'           % spec['mrnet'].prefix,
+                '-DCMAKE_MODULE_PATH=%s'   % join_path(
+                    prefix.share, 'KrellInstitute', 'cmake')]
+
+            # Adjust the standard cmake arguments to what we want the build
+            # type, etc to be
+            self.adjustBuildTypeParams_cmakeOptions(spec, cmake_args)
+
+        else:
+            cmake_args = [
+                '-DBoost_NO_SYSTEM_PATHS=TRUE',
+                '-DXERCESC_DIR=%s'         % spec['xerces-c'].prefix,
+                '-DBOOST_ROOT=%s'          % spec['boost'].prefix,
+                '-DMRNET_DIR=%s'           % spec['mrnet'].prefix,
+                '-DCMAKE_MODULE_PATH=%s'   % join_path(
+                    prefix.share, 'KrellInstitute', 'cmake')]
```

Comment:
Ok.   Looking at these issues.   The if/else is not supposed to be the same.  I guess I put it in as a place holder for the targeting work I thought I would get to quicker than I did.